dictionary/properties/icons/dashboard.yaml

```diff
@@ -9,18 +9,17 @@ icon:
     analytics:
       value: dictionary/assets/icons/dashboard/analytics.svg
 
-    arrow:
-      down:
-        value: dictionary/assets/icons/dashboard/arrow-down.svg
+    arrow-down:
+      value: dictionary/assets/icons/dashboard/arrow-down.svg
 
-      left:
-        value: dictionary/assets/icons/dashboard/arrow-left.svg
+    arrow-left:
```

Comment:
This had the (un/fortunate)? side-effect of needing icons to be explicitly set, otherwise too many subfolders were created and names got split (`dist/icon/arrow/left.svg` vs `dist/icon/arrow-left.svg`)